src/client/components/pages/collection.js

```diff
@@ -298,8 +300,9 @@ class CollectionPage extends React.Component {
 							<Button
 								bsSize="small"
 								bsStyle="warning"
-								className="margin-bottom-d5"
+								className="margin-right-d5"
 								href={`/collection/${this.props.collection.id}/edit`}
+								style={{marginRight: '0.5rem'}}
```

Comment:
The style can be removed, since we're using the class `margin-right-d5` instead.